templates/elasticsearchexporter.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
```

Comment:
that is part of another PR #26 but I am testing it here too.

Reply:
Is this something that we should remove later?

Reply:
no, #26 would go before #27